app/controllers/presentations_controller.rb

```diff
@@ -9,7 +9,12 @@ def new
   end
 
   def create
-    @presentation = Presentation.new(presentation_params)
+    # because of the presentation policy we know a person
+    # will have to be signed in to view this page.
+    # current_person will never be nil
```

Comment:
Yep that's not very clear. I meant you wouldn't be able to view `/presentations/new` without being logged in first.

`current_user` has the potential to be nil if you made the post request from like a ARC or AJAX request outside the app. In that case you would be greeted with a Pundit not authorized error. 

Which is not tested. I'll write up specs for that.